test/utils/image/manifest.go

```diff
@@ -79,55 +85,60 @@ var (
-	CudaVectorAdd            = ImageConfig{e2eRegistry, "cuda-vector-add", "1.0"}
-	Dnsutils                 = ImageConfig{e2eRegistry, "dnsutils", "1.1"}
-	EchoServer               = ImageConfig{e2eRegistry, "echoserver", "2.2"}
-	EntrypointTester         = ImageConfig{e2eRegistry, "entrypoint-tester", "1.0"}
-	Fakegitserver            = ImageConfig{e2eRegistry, "fakegitserver", "1.0"}
-	GBFrontend               = ImageConfig{sampleRegistry, "gb-frontend", "v6"}
-	GBRedisSlave             = ImageConfig{sampleRegistry, "gb-redisslave", "v3"}
-	Hostexec                 = ImageConfig{e2eRegistry, "hostexec", "1.1"}
-	IpcUtils                 = ImageConfig{e2eRegistry, "ipc-utils", "1.0"}
-	Iperf                    = ImageConfig{e2eRegistry, "iperf", "1.0"}
-	JessieDnsutils           = ImageConfig{e2eRegistry, "jessie-dnsutils", "1.0"}
-	Kitten                   = ImageConfig{e2eRegistry, "kitten", "1.0"}
-	Liveness                 = ImageConfig{e2eRegistry, "liveness", "1.0"}
-	LogsGenerator            = ImageConfig{e2eRegistry, "logs-generator", "1.0"}
-	Mounttest                = ImageConfig{e2eRegistry, "mounttest", "1.0"}
-	MounttestUser            = ImageConfig{e2eRegistry, "mounttest-user", "1.0"}
-	Nautilus                 = ImageConfig{e2eRegistry, "nautilus", "1.0"}
-	Net                      = ImageConfig{e2eRegistry, "net", "1.0"}
-	Netexec                  = ImageConfig{e2eRegistry, "netexec", "1.1"}
-	Nettest                  = ImageConfig{e2eRegistry, "nettest", "1.0"}
-	Nginx                    = ImageConfig{dockerLibraryRegistry, "nginx", "1.14-alpine"}
-	NginxNew                 = ImageConfig{dockerLibraryRegistry, "nginx", "1.15-alpine"}
-	Nonewprivs               = ImageConfig{e2eRegistry, "nonewprivs", "1.0"}
-	NoSnatTest               = ImageConfig{e2eRegistry, "no-snat-test", "1.0"}
-	NoSnatTestProxy          = ImageConfig{e2eRegistry, "no-snat-test-proxy", "1.0"}
-	// When these values are updated, also update cmd/kubelet/app/options/container_runtime.go
-	Pause               = ImageConfig{gcRegistry, "pause", "3.1"}
-	Porter              = ImageConfig{e2eRegistry, "porter", "1.0"}
-	PortForwardTester   = ImageConfig{e2eRegistry, "port-forward-tester", "1.0"}
-	Redis               = ImageConfig{e2eRegistry, "redis", "1.0"}
-	ResourceConsumer    = ImageConfig{e2eRegistry, "resource-consumer", "1.4"}
-	ResourceController  = ImageConfig{e2eRegistry, "resource-consumer/controller", "1.0"}
-	ServeHostname       = ImageConfig{e2eRegistry, "serve-hostname", "1.1"}
-	TestWebserver       = ImageConfig{e2eRegistry, "test-webserver", "1.0"}
-	VolumeNFSServer     = ImageConfig{e2eRegistry, "volume/nfs", "1.0"}
-	VolumeISCSIServer   = ImageConfig{e2eRegistry, "volume/iscsi", "1.0"}
-	VolumeGlusterServer = ImageConfig{e2eRegistry, "volume/gluster", "1.0"}
-	VolumeRBDServer     = ImageConfig{e2eRegistry, "volume/rbd", "1.0.1"}
+	// PrivateRegistry is a user configured image repository
```

Comment:
this is not quite correct. PrivateRegistry is an image repository that requires authentication.

Reply:
updated, thanks for the review :)